examples/routing_service/routing_service_msql_adapter/cpp/rs_dll/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+#
+# (c) 2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+
+
+cmake_minimum_required(VERSION 3.11)
+project (MsqlAdapter)
+
+if(DEFINED ENV{NDDSHOME})
+	# NDDSHOME is set
+else()
+    set(msg
+        "NDDSHOME environment variable must be set")
+    message(FATAL_ERROR ${msg})
+endif()
```

Comment:
This is not needed. Please, could you remove it?

Reply:
Done